src/app/components/footer/footer.component.html

```diff
@@ -0,0 +1,48 @@
+<footer class="footer">
+  <div class="footer__content">
+    <div class="footer__logo">
+      <img src="assets/logo.png" alt="Logo de Jean-Luc Laffuge" />
+      <h4 class="footer__title">Cinq de cœur</h4>
+    </div>
+
+    <div class="footer__contact">
+      <h4 class="footer__subtitle">Contact</h4>
+      <a class="footer__text" [href]="'https://www.google.com/maps/search/' + encodeAddress(contactDetails.address)" target="_blank" rel="noopener noreferrer">
+        {{ contactDetails.address }}
+      </a>
+      <a class="footer__text" [href]="'tel:' + contactDetails.phone">
+        {{ contactDetails.phone }}
+      </a>
+      <a class="footer__text" [href]="'mailto:' + contactDetails.email">
+        {{ contactDetails.email }}
```

Comment:
We need to remove it to !

Reply:
I've encoded in Base64, let me know is it's ok for you !

Reply:
⬆️ Same ! ⬆️